Person.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GetterSetterAssignment
+{
+    public class Person
+    {
+        public string? Name;

```

Comment:
Any specific reason why we are making name nullable and not age.

Reply:
No specific reason , it was just to accept null values.